lgc/include/lgc/state/AbiMetadata.h

```diff
@@ -94,6 +94,9 @@ namespace PipelineMetadataKey {
 static constexpr char Name[] = ".name";
 static constexpr char Type[] = ".type";
 static constexpr char InternalPipelineHash[] = ".internal_pipeline_hash";
+static constexpr char XglCacheInfo[] = ".xgl_cache_info";
```

Comment:
@trenouf Let me know if you want a different layout or names for these entries.

Reply:
I'll check with Rob.

Reply:
Rob said this was fine.